packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -46,13 +46,18 @@ export const ChatMessageComponentAsMessageBubble = (props: ChatMessageComponentA
     React.MutableRefObject<HTMLElement | null> | undefined
   >(undefined);
 
+  // Track if the action menu was opened by touch - if so we increase the touch targets for the items
+  const actionFlyoutLastOpenedByTouch = useRef(false);
+
   const chatActionsEnabled = !disableEditing && message.status !== 'sending' && !!message.mine;
   const actionMenuProps = chatMessageActionMenuProps({
     enabled: chatActionsEnabled && allowChatActionButtonShow,
     // Force show the action button while the flyout is open and targeting the action menu button
     forceShow: chatMessageActionFlyoutTarget === messageActionButtonRef,
     menuButtonRef: messageActionButtonRef,
     onActionButtonClick: () => {
+      actionFlyoutLastOpenedByTouch.current = false;
```

Comment:
Do we also consider show the contextual menu when user long click the chat bubble? (Maybe that flyout is not necessary for touch?)

Reply:
That is done here: https://github.com/Azure/communication-ui-library/pull/1020

Reply:
Looks like it is already working as what I said, nice!